ART - Communities presentation dialog and refactoring.

GaboHub opened the pull request:
The communities presentation dialog is working correctly and ready to test.

darkestpriest commented:
This import `import com.bitbudai.fermat_cht_android_sub_app_chat_bitdubai.util.CommonLogger;`
doesn't exists in the code, please, push this class.
Also, this pull-request has conflicts

GaboHub committed:
Merge branch 'develop' of https://github.com/darkestpriest/fermat into develop

GaboHub committed:
Merge branch 'develop' of https://github.com/darkestpriest/fermat into develop

GaboHub committed:
Merge branch 'develop' of https://github.com/darkestpriest/fermat into develop

GaboHub committed:
Merge branch 'develop' of https://github.com/darkestpriest/fermat into develop

GaboHub committed:
Merge branch 'develop' of https://github.com/darkestpriest/fermat into develop

GaboHub committed:
Merge branch 'develop' of https://github.com/darkestpriest/fermat into develop

GaboHub committed:
Refactor: ART Fan and Artist sup App community

Deleted unsued android resources. fixing resources name.

Include in: #5977

Signed-off-by: Gabriel Araujo <gabe_512@hotmail.com>

GaboHub committed:
Fix: Fan And Artist - Method getSelectedActorIdentity

Added some extra logic at the moment to throw ActorIdentityNotSelectedException and CantGetSelectedActorIdentityException.
In the world fragment added a dialog without identites.

Include in: #6053

Signed-off-by: Gabriel Araujo <gabe_512@hotmail.com>

GaboHub committed:
Merge branch 'develop' of https://github.com/darkestpriest/fermat into develop
